Sources/TokamakCore/App/Scenes/WindowGroup.swift

```diff
@@ -15,6 +15,11 @@
 //  Created by Carson Katri on 7/16/20.
 //
 
+@_spi(TokamakCore)
+public struct _WindowGroupTitle: _PrimitiveView {
+  public let title: Text?
```

Comment:
Does this need to be an optional?

In `WindowGroup`s where `title` is nil, it should be more efficient to not use a `_WindowGroupTitle` at all.